src/app/pages/send/send-crypto-asset-form/send-crypto-asset-form.routes.tsx

```diff
@@ -69,7 +68,7 @@ export const sendCryptoAssetFormRoutes = (
       {ledgerTxSigningRoutes}
       {recipientAccountsDrawerRoute}
     </Route>
-    <Route path="/send/:symbol/confirm" element={<StacksSip10SendFormConfirmation />} />
+    <Route path="/send/:symbol/confirm" element={<StxSendFormConfirmation />} />
```

Comment:
If we are now handling both `STX` and other sip10 tokens together we should use the word `Stacks` rather than the token symbol `Stx` here.